sdk/eventgrid/eventgrid-systemevents/src/cadl-generated/logger.ts

```diff
@@ -2,4 +2,4 @@
 // Licensed under the MIT License.
 
 import { createClientLogger } from "@azure/logger";
-export const logger = createClientLogger("eventgrid-system-events");
+export const logger = createClientLogger("eventgrid-systemevents");
```

Comment:
Not blocking for this PR but would it make sense to rename `cadl-generated` to just `generted` (or even `typespec-generated`) just to remove any cadl reference? Could make a small fixit issue for it

Reply:
I am going to do an update for this month. Will do the rename as part of that.